lib/Authentication/AuthTokens/ShibAuthToken.php

```diff
@@ -204,4 +204,27 @@ public static function isStateless() {
         return true;
     }
 
+    private function getEntitlementErrorMessage()
```

Comment:
Can you settle on double-quotes throughout this method?

Reply:
Sure Greg. Done